.pre-commit-config.yaml

```diff
@@ -56,7 +56,7 @@ repos:
         exclude: "CHANGELOG.md"
 
   - repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
-    rev: v9.11.0
+    rev: v9.13.0
```

Comment:
Hi @DSuveges, I was already afraid this would cause some trouble. I am happy to revert, but as mentioned, when running `make setup-dev` this file is automatically updated. If I revert the updates pre-commit fails and I am unable to do any commits...

Trying to revert the specific commit:
![image](https://github.com/opentargets/gentropy/assets/76998782/c6b2f468-2c5e-4acf-941f-a9e390a3b988)

Running `make setup-dev`: (notice the updating of ruff and commitlint)
![image](https://github.com/opentargets/gentropy/assets/76998782/9d53ae75-a73d-4f8e-8b0a-b5bf9059b21e)

Reply:
Hi @DSuveges,

I have updated our `dev` branch to include https://github.com/opentargets/gentropy/pull/524. I re-created the development environment by running `make setup-dev` but keep running into this error.

![image](https://github.com/opentargets/gentropy/assets/76998782/29af4637-2a8a-41f5-b5cc-af29553dedfe)

May I ask which `node` version you are using?